subset/dot1x/Dockerfile.authenticator

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
```

Comment:
This should be based on aardvark or aardvark2, helps to amortize the build cost.

Reply:
Makes sense. Changed.